src/renderers/WebGLRenderer.js

```diff
@@ -821,7 +832,7 @@ function WebGLRenderer( parameters ) {
 
 	function setupVertexAttributes( material, program, geometry ) {
 
-		if ( geometry && geometry.isInstancedBufferGeometry & ! capabilities.isWebGL2 ) {
+		if ( geometry && geometry.isInstancedBufferGeometry && ! capabilities.isWebGL2 ) {
```

Comment:
I think we you rebase to the latest commit, this modification should go away.

Reply:
@Mugen87 Got it. Thanks!